app/models/order.rb

```diff
@@ -1,7 +1,17 @@
+require 'date'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.